app/assets/stylesheets/article-show.scss

```diff
@@ -41,6 +41,11 @@ article {
   }
 }
 
+#bottom-content-read-next {
+  content-visibility: auto;
+  contain-intrinsic-size: 450px;
```

Comment:
Interesting, this is a new one for me. How did we decide on/determine on the contain-intrinsic-size pixel value (not just here but across this PR?)

Reply:
I just eyeballed it and picked a number based on what I think the average space the titles might take up. Anything that includes user-generated content can't be exact, but this value is just designed to be an estimate. I figured this area (four titles) won't have as much variation as, say, a comment section where each comment could be _any_ height.

Reply:
I imagine we need a media query with a different `contain-intrinsic-size` for smaller screens/devices? Same for all the places below where we use `contain-intrinsic-size`.

Reply:
I just looked at this `<section>` on DEV and I worry that the hardcoded pixel value doesn't match what the element will actually look like:
<img width="1532" alt="Screen Shot 2020-11-23 at 9 10 02 AM" src="https://user-images.githubusercontent.com/6921610/99992886-c97b8c80-2d6b-11eb-9139-26c3b65eedad.png">

+1 to @nickytonline's point of accounting for the different sizes at various screen sizes here.

Reply:
My interpretation of how to use `contain-intrinsic-size` is that anything in the ballpark of the ultimate size is fine, so loose estimates are accurate enough. We wouldn't want a _radically_ bad guess (aka comment height could be off by an order of magnitude) but if we're giving the browser a hint it will still be an additive feature.

It's hard to get a 100% obvious answer to this question, but I think that's the case. And if it is the case, the divs I chose happen to be losely similar heights regardless of viewport width.

The word "estimate" is generally used everywhere alongside this feature (though definitely not enough written about this for us to have 100% confidence)

From: https://www.w3.org/TR/css-sizing-4/#cis-scrollbars

> The contain-intrinsic-size property provides an estimate of how large the author expects the content of an element to be, but this estimate is not actual content and does not represent anything that needs to be shown to the user.

Let me know if my interpretation seems offbase.

Reply:
Based on what you're saying it sounds fine. I've never used this feature so we can always try it out.

Reply:
Ok, that's helpful—the additional link you provided makes it seem like it's safe enough to have a loose estimate here. If we notice any weird behavior because of this, we can always change it.